test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -231,12 +263,8 @@ public function invalidVersionStringsProvider() : array
     {
         return [
             [''],
-            ['12.a'],
```

Comment:
`.a` is not a valid stability provider, innit?

Reply:
Same for the data provider changes below: are we sure these are valid now?

Reply:
`a` is short for alpha ? honestly I just stole that regExp from semver ) https://github.com/composer/semver/blob/2b303e43d14d15cc90c8e8db4a1cdb6259f1a5c5/src/VersionParser.php#L39

I tried to do a bit of research on what kind of version are correct what are the types of versioning and so on.. didn't help much